src/services/organizeImports.ts

```diff
@@ -135,6 +143,14 @@ namespace ts.OrganizeImports {
         }
     }
 
+    function hasModuleDeclarationMatchingSpecifier(sourceFile: SourceFile, moduleSpecifier: Expression) {
+        const moduleSpecifierText = isStringLiteral(moduleSpecifier) && moduleSpecifier.text;
+        return isString(moduleSpecifierText) && some(sourceFile.statements, statement =>

```

Comment:
Also may be use `sourceFile.moduleAugmentations` ?

Reply:
❗️didn't know this exists, thanks!